docs/content/en/docs-v0.46.x/installation/install-control-plane/installing-controlplane-on-k8s.md

```diff
@@ -72,7 +74,7 @@ helm upgrade -i pipecd oci://ghcr.io/pipe-cd/chart/pipecd --version {{< blocks/l
   --set-file secret.gcsServiceAccount.data=path-to-service-account-file
 ```
 
-Currently, besides `Firestore` PipeCD supports other databases as its datastore such as `MySQL`. Also as for filestore, PipeCD supports `AWS S3` and `MINIO` either.
+#### Using MySQL and MINIO
```

Comment:
How about using the tab switchable configuration like this?
ref: https://github.com/pipe-cd/pipecd/blob/master/docs/content/en/docs-dev/installation/install-piped/installing-on-kubernetes.md?plain=1#L46-L86

https://github.com/pipe-cd/pipecd/assets/32532742/0270e01c-a246-4965-88ea-9060db6c2bdb

Reply:
You can see how it looks here: https://pipecd.dev/docs-v0.47.x/installation/install-piped/installing-on-kubernetes/#:~:text=the%20full%20configuration.-,Installing%20by%20using,-Helm%20(3.8.0%20or